http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/DiskServiceInterp.scala

```diff
@@ -318,7 +318,7 @@ class DiskServiceInterp[F[_]: Parallel](config: PersistentDiskConfig,
       disks <- listDisks(
         userInfo,
         Some(cloudContext),
-        Map.empty
+        Map(includeDeletedKey -> "true")
       )
       _ <- disks.traverse(disk => deleteDiskRecords(userInfo, cloudContext, disk))
```

Comment:
Just thinking out loud, in deleteDiskRecords, there's a call to `persistentDiskQuery.delete`, which would be redundant if the Leo record is already set to 'deleted' and would overwrite the original `destroyedDate`. Not a huge deal, but maybe something to test to make sure that's working correctly in the case that Leo has already "deleted" the resource